insert-cover-pages-example/src/parameters.ts

```diff
@@ -0,0 +1,25 @@
+export class Parameters {
```

Comment:
I would not create this class at all. It does not save much and after changes ("as const") in parameters will have correct types from start.

Reply:
Parameters class was changed to interface... I´m not sure if it´s best solution, but imho better than access via context.